server/seeder.test.js

```diff
@@ -0,0 +1,27 @@
+const seed = require('./seeder')
+
+test('the movie list has given movie', async() => {
+  expect(seed.movieNames).toContain('Amelie');
```

Comment:
This is not a very useful test, and I would recommend getting rid of it. (I think it's at high risk of becoming out of date and doesn't really give us more confidence in how our code is behaving.)